hyperopt/fmin.py

```diff
@@ -543,6 +548,9 @@ def fmin(
             raise Exception(
                 "There are no evaluation tasks, cannot return argmin of task losses."
             )
+        if pickle_return is not None:
+            with open(pickle_return + '.pkl', 'wb') as out_file:
```

Comment:
please black this

Reply:
Could you change this to force `pickle_return` to be a string to avoid errors.  So line 552 would be:
```suggestion
            with open(str(pickle_return) + '.pkl', 'wb') as out:file:
```
I used the black [playground](https://black.now.sh) and my stuff came back the same way I put it in.  I have never used black before.